it/uk/gov/hmrc/agentmappingfrontend/controllers/FieldMappingsSpec.scala

```diff
@@ -19,11 +19,11 @@ class FieldMappingsSpec extends UnitSpec with EitherValues {
     }
 
     "give \"error.required\" error when it is empty" in {
```

Comment:
"error.required" in the test name is now "error.utr.blank". Same deak with the next test below it